tensorflow/python/ops/math_ops.py

```diff
@@ -130,8 +134,8 @@ def abs(x, name=None):  # pylint: disable=redefined-builtin
     name: A name for the operation (optional).
 
   Returns:
-    A `Tensor` or `SparseTensor` the same size and type as `x` with absolute
-      values.
+    A `Tensor` or `SparseTensor` the same size, type, and sparsity as `x` with 
+      absolute values. 
```

Comment:
nit, remove space

Reply:
Removed, thanks for catching that. Note to self: Run CI tests on documentation branches :)